InjectionByExample.Tests/ContainerBuilderTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using InjectionByExample;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace InjectionByExample.Tests
+{
+    [TestClass()]
+    public class ContainerBuilderTests
+    {
+        ContainerBuilder containerBuilder = new ContainerBuilder();
+
+        [TestMethod()]
+        public void RegisterOneGenericType()
+        {
+            containerBuilder.Register<Driver>();
+
+            Type t = containerBuilder.GetType();
+            Assert.IsInstanceOfType(containerBuilder, t);
+        }
+
+        [TestMethod()]
+        public void RegisterTwoGenericType()
+        {
+            containerBuilder.Register<Driver, Driver>();
+
+            Type t = containerBuilder.GetType();
```

Comment:
I think this test mostly verifies the GetType() method rather than anything of the target code

Reply:
You're right, it really was useless to just check the type. It was a lack of imagination. I changed it and I think it's better.